web/views_whiteboard.py

```
@@ -0,0 +1,212 @@
+    """
+    Live whiteboard view for a specific virtual classroom/course.
+    """
+    # Get the classroom or course
+    classroom = None
+    course = None
+    
+    # First try to get by classroom_id
+    try:
+        classroom = get_object_or_404(VirtualClassroom, id=classroom_id)
+        course = classroom.course
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)